packages/FormElement/stories/examples/Everything.js

```diff
@@ -15,34 +15,38 @@ export default function EverythingExample() {
   const [value2, setValue2] = React.useState();
   const [errorText, setErrorText] = React.useState();
 
-  const helpContent = (
-    <>
-      Help yourself{" "}
-      <a href="https://youtu.be/D-Zy2P26gVM" target="_blank" rel="noopener noreferrer">
-        learn more
-      </a>
-      .
-    </>
+  const LabelWithToolTip = ({ children }) => (
```

Comment:
This is more DRY, but maybe not as clear for a Storybook example, where it might be nicer to avoid the indirection and just have the actual `<Label>` in line with the rest of the JSX.  What do you think?